File: Famoser.MassPass.Api/src/Models/Request/Authorization/UnAuthorizationRequest.php
<?php
/**
 * Created by PhpStorm.
 * User: famoser
 * Date: 22/05/2016
 * Time: 22:57
 */

namespace Famoser\MassPass\Models\Request\Authorization;


use Famoser\MassPass\Models\Request\Base\ApiRequest;

class UnAuthorizationRequest extends ApiRequest
{
    public $DeviceToBlockId;
    public $Reason;
}

File: Famoser.MassPass.Data/Entities/Communications/Request/Authorization/UnAuthorizationRequest.cs
using System;
using Famoser.MassPass.Data.Entities.Communications.Request.Base;

namespace Famoser.MassPass.Data.Entities.Communications.Request.Authorization
{
    public class UnAuthorizationRequest : ApiRequest
    {
        public Guid DeviceToBlockId { get; set; }
        public string Reason { get; set; }
    }
}
